lib/src/json_document_transformer.dart

```diff
@@ -35,9 +36,10 @@ class JsonDocumentTransformer
 
   JsonDocumentTransformer._(this._codec);
 
-  StreamChannel bind(StreamChannel<String> channel) {
+  StreamChannel<Object> bind(StreamChannel<String> channel) {
     var stream = channel.stream.map(_codec.decode);
-    var sink = new StreamSinkTransformer.fromHandlers(handleData: (data, sink) {
+    var sink = new StreamSinkTransformer.fromHandlers(
+        handleData: (Object data, EventSink<String> sink) {
```

Comment:
Style nit: I generally tend to prefer adding type parameters to the function rather than to callbacks being passed in to the function (e.g. `new StreamSinkTransformer<Object, String>.fromHandlers(...)`). It saves on tokens (you don't have to write `EventSink` explicitly) and makes it more obvious what the return type is.

Reply:
done